.editorconfig

```diff
@@ -4,6 +4,7 @@
 
 root = true
 
+
```

Comment:
Seems vaguely odd, I didn't realize we had two newlines back to back in this file...

Reply:
https://github.com/ember-cli/ember-cli/pull/9013 😉